tealer/teal/instructions/parse_instruction.py

```diff
@@ -146,13 +146,20 @@ def handle_gtnxa(x: str) -> instructions.Gtxna:
 
 
 def parse_line(line: str) -> Optional[instructions.Instruction]:
+    comment = ""
     if "//" in line:
-        line = line[0 : line.find("//")]
+        comment_start = line.find("//")
+        # save comment
+        comment = line[line.find("//") : len(line)]
+        # strip line of comments and leading/trailing whitespace
+        line = line[0 : comment_start].strip()
```

Comment:
```suggestion
        line = line[:comment_start].strip()
```

Reply:
Again, sorry for my lack of knowledge on Python idioms.